src/app/components/LeftMenu/PartyCardList/index.tsx

```diff
@@ -16,11 +18,33 @@ const Box = styled.div`
     background: transparent;
 `;
 
+function score(party, search){
```

Comment:
score function should be work as model layer.

I think redux layer maybe fit to purpose....